### ISSUE TITLE
Items get deleted when thrown into a weapon rack

### ISSUE THREAD

**Issue by glennmayo** [Storkman101/Void-World#14]
Hello whoever reads this, I accidentally closed my inventory when I had the quest item Researcher Ernest clicked on so it flew out and onto a weapon rack. It instantly deleted and I know those kind of bugs I should send to the mod author but I'm posting it here because other blocks may do that and you could prevent future problems by possibly putting extras somewhere or having them spawn in when you step on a pressure plate and activate a command block. Something along those lines I guess. Well thanks anyway the pack is really awesome I had a lot of fun playing it and I will definitely continue playing in the future.

**Comment by Storkman101** [comment]
This has something to do with all the quest gate systems there are there. I tested it myself (in a new version tho with a newer version of the mod) it just gets thrown behind you or outside. Depends a bit how the luck is. Or it floats in mid air. Have not been able to throw it and make it dissapear in the newest version. So or it has been fixed in the newer version or you didnt saw it floating somewhere in the air or behind you
